mis_builder/models/aep.py

```diff
@@ -67,9 +67,11 @@ class AccountingExpressionProcessor(object):
                          r"(?P<accounts>_[a-zA-Z0-9]+|\[.*?\])"
                          r"(?P<domain>\[.*?\])?")
 
-    def __init__(self, company):
-        self.company = company
-        self.dp = company.currency_id.decimal_places
+    def __init__(self, companies, currency_id=None):
+        self.companies = companies
+        self.currency_id = currency_id and currency_id or \
+            self.companies.env.user.company_id.currency_id
```

Comment:
In that case the user will specify the currency explicitly in mis_report_instance.
AEP is a low level API, and I want it to have as little implicit behaviour as possible.